### ISSUE TITLE
Build failure: termscp

### ISSUE THREAD

**Issue by nyabinary** [NixOS/nixpkgs#353401]
### Steps To Reproduce

1. build termscp

### Build log

```
error: builder for '/nix/store/mhli87afh2lag9izd8sfwy2fafzh7fcr-termscp-0.15.0.drv' failed with exit code 101;
       last 25 log lines:
       >    Compiling nucleo v0.5.0
       >    Compiling argh v0.1.12
       >    Compiling magic-crypt v3.1.13
       >    Compiling remotefs-ftp v0.2.0
       >    Compiling keyring v3.3.0
       >    Compiling tui-realm-stdlib v1.3.2
       >    Compiling remotefs-smb v0.3.0
       >    Compiling remotefs-ssh v0.4.0
       >    Compiling toml v0.8.19
       >    Compiling remotefs-kube v0.4.0
       >    Compiling remotefs-aws-s3 v0.3.0
       >    Compiling notify-rust v4.11.3
       >    Compiling hostname v0.4.0
       >    Compiling content_inspector v0.2.4
       >    Compiling version-compare v0.2.0
       >    Compiling whoami v1.5.2
       >    Compiling unicode-width v0.2.0
       >    Compiling bytesize v1.3.0
       > error: unexpected `--cfg unix` flag
       >   |
       >   = note: config `unix` is only supposed to be controlled by `--target`
       >   = note: manually setting a built-in cfg can and does create incoherent behaviors
       >   = note: `#[deny(explicit_builtin_cfgs_in_flags)]` on by default
       >
       > error: could not compile `termscp` (bin "termscp") due to 1 previous error
       For full logs, run 'nix log /nix/store/mhli87afh2lag9izd8sfwy2fafzh7fcr-termscp-0.15.0.drv'.
error: 1 dependencies of derivation '/nix/store/1cqgnl79r75nf3n7x2qzyibjd9p4qwhj-user-environment.drv' failed to build
warning: killing stray builder process 121598 ()...
warning: killing stray builder process 121737 ()...
warning: killing stray builder process 126364 ()...
warning: killing stray builder process 135878 ()...
warning: killing stray builder process 136275 ()...
warning: killing stray builder process 136276 ()...
warning: killing stray builder process 136548 ()...
warning: killing stray builder process 136556 ()...
```

### Additional context
NixOS unstable branch

### Notify maintainers
@fabaff 

### Metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.11.5, NixOS, 24.11 (Vicuna), 24.11.20241029.807e915`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.25.0pre20240920_ca3fc169`
 - nixpkgs: `/nix/store/m68ikm8045fj7ys7qvgr608z9l70hh1k-source`
---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by samueltardieu** [comment]
That should be fixed by #349034

**Comment by nyabinary** [comment]
> That should be fixed by #349034

It won't it needs to be updated to this commit: https://github.com/veeso/termscp/commit/e45c3d5b4ef64653e5b6cc4f3703e3b67514306d
